_includes/configuration/tabfocus_elements.md

```diff
@@ -0,0 +1,23 @@
+### `tabfocus_elements`
+
+This option enables you to specify an element `ID` to focus when the user presses the tab key inside the editor. You can also use the special `":prev"` and `":next"` values. It will then place the focus on either the previous or next input element placed before/after the {{site.productname}} instance in the DOM.
```

Comment:
Sigh, more pronouns :cry:

Reply:
yeah, pls ignore for this PR :disappointed: